### DISCUSSION TITLE
Export function

### DISCUSSION THREAD

**Discussion by samearcher** [jowilf/starlette-admin#544] [Q&A]
Is it possible to enable export of ALL rows of a table, not just those showing in the list view?

Thanks

Sam

**Comment by hasansezertasan** [comment]
> Is it possible to enable export of ALL rows of a table, not just those showing in the list view?
> 
> Thanks
> 
> Sam

I believe you can add such a functionality to each model view. I already proposed such a feature at #390, let's call it `view_actions`. And the out-of-box examples of this feature could be `export_all` and `import_many` (#479).

But I am not the maintainer/author. We have to wait for the author's response.
